Packs/TAXIIServer/Integrations/TAXII2Server/TAXII2Server.py

```diff
@@ -934,7 +931,7 @@ def create_extension_definition(object_type, extensions_dict, xsoar_type,
             'version': '1.0',
             'extension_types': ['property-extension']
         }
-        extensions_dict[object_type] = {'extension_id': extension_id, 'xsoar_indicator_to_return': xsoar_indicator_to_return}
+        extensions_dict[object_type] = True
```

Comment:
whats this change?

Reply:
Since I don't really need to keep track of the values, this is enough to assign a boolean value to flag that his object_type already has an extension definition.